paper.md

```diff
@@ -2,11 +2,77 @@
 title: "Popper 2.0: A Container-Native Workflow Execution Engine For Testing Complex Applications and Reproducing Scientific Explorations"
 author: Ivo Jimenez, Jayjeet Chakraborty, Arshul Mansoori, Quincy Wofford and Carlos Maltzahn
 abstract: |
- Abstract
+    The problem of reproducibility and replication in scientific research is quite prevalent to date. 
+    Researchers working in fields of computational science often find it difficult to reproduce experiments from artifacts like code, data, diagrams and results which are left behind by the previous researchers. 
+    The code developed on one machine often fails to run on other machines due to differences in hardware architecture, OS, software dependencies, among others. 
+    This is accompanied by the difficulty in understanding how artifacts are organized, as well as in using them in correct order. 
+    Since this problem of platform dependency can be solved by using Software containers (light weight virtualization), researchers and developers have built scientific workflow engines like Argo, Pachyderm, Brigade, etc. for organizing the steps in a workflow as Directed Acyclic Graphs and running them in separate containers.
+    But these existing container-native workflow engines assume the availability of a Kubernetes cluster deployed in the cloud, access to which is not always trivial.
+    Therefore, there is a need for a container-native workflow engine that does not assume the presence of a Kubernetes cluster or any other specific computing environment.
+    In this paper, we introduce Popper, a container-native workflow engine that executes each step of a workflow in a separate dedicated container and does not assume the presence of a Kubernetes cluster or any cloud based Kubernetes service.
+    With Popper, researchers can build and validate workflows easily in almost any environment of their choice like a local machine, a Slurm based HPC cluster or a Kubernetes based cloud computing environment. 
+    To exemplify the suitability of this workflow engine, we present three case studies where we take examples from Machine Learning and HPC and turn them into Popper workflows.
```

Comment:
Before this sentence, we can also say that this paper presents the design and architecture of the engine and explains how we get to abstract workflow execution over multiple resource managers and environments